toptal/picasso
# fix: shadows are broken in Safari & iOs

[FX-2237]

### Description

Fix for bug in the Safari on Mac OS & mobile iOs.
Happens due to transparent doesn't work well with gradient.

### How to test

- Open `Safari` app on Desktop
- Visit [modal component](https://picasso.toptal.net/fx-fix-modal-contend-shadow-when-overflow/?path=/story/overlays-modal--modal)
- Click `Edit Code` next to `Default`
- Duplicate form to make sure there will be content overflow
- **Result**: shadow looks good 👍 

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="1440" alt="Screenshot 2021-11-08 at 18 55 45" src="https://user-images.githubusercontent.com/5591973/140784698-38c43eaa-49ad-4077-8d81-820fbedfc180.png"> | <img width="1440" alt="Screenshot 2021-11-08 at 18 54 20" src="https://user-images.githubusercontent.com/5591973/140784482-2f2bc390-3cc1-4abd-8ae5-968309c7b9a7.png"> |


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-2237]: https://toptal-core.atlassian.net/browse/FX-2237?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
